ansible_collections/arista/avd/roles/eos_designs/python_modules/interface_descriptions/avdinterfacedescriptions.py

```diff
@@ -58,7 +58,7 @@ def underlay_ethernet_interfaces(self, link_type: str, link_peer: str, link_peer
             )
 
         link_peer = str(link_peer).upper()
-        if link_type == "underlay_p2p":
+        if link_type in ["underlay_p2p", "underlay_p2p_vrfs"]:
```

Comment:
underlay_p2p_vrfs is a bit misleading. The VRFs are in the "overlay" (sort of). Just use "p2p_vrfs"